pkg/sql/opt/exec/execbuilder/testdata/partial_index

```diff
@@ -222,24 +222,24 @@ query T kvtrace
 INSERT INTO t VALUES (5, 4, 'bar')
 ----
 CPut /Table/112/1/5/0 -> /TUPLE/2:2:Int/4/1:3:Bytes/bar
-InitPut /Table/112/2/4/5/0 -> /BYTES/
```

Comment:
It's not obvious to me why these were `InitPut`s in the first place - these secondary index keys should not already exist. Was there a performance benefit to only using `CPut` for the primary index to catch conflicts? Does the same benefit still exist for `CPut` vs `CPutAllowingIfNotExists`? If so, should we differentiate between the two types of `CPut`s int traces like this? And if there is no benefit to`CPut` over `CPutAllowingIfNotExists`, should we convert these secondary index puts into `CPut`s?